docs/usage/configuration/config_documentation.md

```diff
@@ -3014,6 +3014,12 @@ Options for each entry include:
      which is set to the claims returned by the UserInfo Endpoint and/or
      in the ID Token.
 
+* `backchannel_logout_enabled`: set to true to process OIDC Back-Channel Logout notifications. 
+  Those notifications are expected to be received on `/_synapse/client/oidc/backchannel_logout`.
+
+* `backchannel_logout_ignore_sub`: by default, the OIDC Back-Channel Logout feature checks that the
+  `sub` claim matches the subject claim received during login. This check can be disabled by setting
+  this to `false`.
```

Comment:
Why would you disable this?

Reply:
There are (rare) cases where you want to use something else as `subject_claim`, like the entity UUID in your upstream LDAP or something like that. In those cases, the `external_id` we save for the user does not match the `sub` claim.

There is a warning on startup to check that